The web container cannot be built due to #11356.

### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

upstream main branch

### Cloud or Self Hosted

Self Hosted (Docker), Self Hosted (Source)

### Steps to reproduce

1. Checkout the latest `main` branch
2. `cd web`
3. `docker built -t web .`

### ✔️ Expected Behavior

`web` image can be built.

### ❌ Actual Behavior

```bash
 => ERROR [builder 4/4] RUN yarn build                                                                                                                                     65.9s
------                                                                                                                                                                           
 > [builder 4/4] RUN yarn build:                                                                                                                                                 
0.432 yarn run v1.22.22                                                                                                                                                          
0.473 $ next build                                                                                                                                                               
1.897 Attention: Next.js now collects completely anonymous telemetry regarding usage.                                                                                            
1.898 This information is used to shape Next.js' roadmap and prioritize features.                                                                                                
1.899 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
1.899 https://nextjs.org/telemetry
1.899 
1.985   ▲ Next.js 14.2.10
1.985 
2.167    Creating an optimized production build ...
65.58 Failed to compile.
65.58 
65.58 ./app/components/workflow/hooks/use-nodes-layout.ts
65.58 Module not found: Can't resolve 'elkjs/lib/elk.bundled.js'
65.58 
65.58 https://nextjs.org/docs/messages/module-not-found
65.58 
65.58 Import trace for requested module:
65.58 ./app/components/workflow/hooks/index.ts
65.58 ./app/components/workflow/index.tsx
65.58 ./app/(commonLayout)/app/(appDetailLayout)/[appId]/workflow/page.tsx
65.58 
65.58 
65.58 > Build failed because of webpack errors
65.80 error Command failed with exit code 1.
65.80 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
------
Dockerfile:30
--------------------
  28 |     COPY . .
  29 |     
  30 | >>> RUN yarn build
  31 |     
  32 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c yarn build" did not complete successfully: exit code: 1
```

### Additional Information

The job on GHA also failed due to the same reason: https://github.com/langgenius/dify/actions/runs/12179987133/job/33973331093

F.Y.I. @charli117 @crazywoola

--- a/web/package.json
+++ b/web/package.json
@@ -52,6 +52,7 @@
     "dayjs": "^1.11.7",
     "echarts": "^5.4.1",
     "echarts-for-react": "^3.0.2",
+    "elkjs": "^0.9.3",
     "emoji-mart": "^5.5.2",
     "fast-deep-equal": "^3.1.3",
     "i18next": "^22.4.13",

